resources/perform_rate_limiting.lua

```diff
@@ -20,10 +20,11 @@ end
 
 local storedEpoch = redis.call('get', epochKey)
 local currentTokens = 0
-if storedEpoch == false or (currentEpoch - tonumber(storedEpoch)) > (secondsInMinute * millisecondsInSecond) then
-    redis.call('set', epochKey, currentEpoch, 'EX', keyTimeout)
+if storedEpoch == false or currentEpoch > tonumber(storedEpoch) then
```

Comment:
should we call it something more specific than `storedEpoch` to denote that it's the time when the rate limit resets?